printrun/gviz.py

```diff
@@ -292,17 +292,17 @@ def zoom(self, x, y, factor):
 
         self.translate = [x - (x - self.translate[0]) * factor,
                           y - (y - self.translate[1]) * factor]
-        penwidth = max(1.0, self.filament_width * ((self.scale[0] + self.scale[1]) / 2.0))
+        penwidth = max(1, int(self.filament_width * ((self.scale[0] + self.scale[1]) / 2.0)))
         for pen in self.penslist:
             pen.SetWidth(penwidth)
```

Comment:
I'd recommend keeping penwidth as float and doing pen.SetWidth(int(penwidth)) instead of this, for consistency with all the others where you convert to integer at the point of call.

Reply:
I haven't done that mostly because:

https://github.com/kliment/Printrun/blob/5a328ff8a64bf25532ed866d85c74b1d42def6d0/printrun/gviz.py#L195-L201

Do you prefer if I add `int()` to all those lines?

Reply:
No, you're entirely right to do it as you did. It's a bit inconsistent but better to have it in one place than many.